_posts/2015-03-23-aerogear-ups-103.md

```diff
@@ -0,0 +1,25 @@
+---
+layout: news
+section: news
+title: AeroGear UnifiedPush Server 1.0.3 released
+author: matzew
+module: push
+releases: 1.0.3
+---
+
+The AeroGear team is happy to announce the version 1.0.3 of the UnifiedPush Server! The release focuses on stabilization and adds a few new features
+
+* Migration support for 1.0.x users as explained in our [user guide](https://aerogear.org/docs/unifiedpush/ups_userguide/index/#migration-guide)
```

Comment:
Is migration a feature?

If yes, I would start with the features that caused a need for migration and then explained they unfortunately requires database migration.